### ISSUE TITLE
Make footer links respond to focus event visibly

### ISSUE THREAD

**Issue by crowbot** [mysociety/fixmytransport#269]
Feedback from Steve:

While navigating page http://www.fixmytransport.com/?beta=1using keyboard only (tab), I was unable to see focus state for links at the bottom of the page: Built by mySociety, which is supported by the Indigo Trust, the Esmée Fairbairn Foundation, the Omidyar Network and many small donors.
